backends/arm/_passes/conv1d_unsqueeze_pass.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# Copyright 2024 Arm Limited and/or its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import torch
+from executorch.backends.arm._passes.arm_pass_utils import (
+    create_node,
+    get_param_tensor,
+    insert_q_dq_pair,
+    is_param_node,
+)
+from executorch.backends.arm.tosa_quant_utils import dq_op, q_op
+from executorch.exir import ExportedProgram
+from executorch.exir.dialects._ops import ops as exir_ops
+from executorch.exir.pass_base import ExportPass, PassResult
+
+
+class Conv1dUnsqueezePass(ExportPass):
```

Comment:
can we use xnnpack's pass here somehow? OK with duplication TBH if it means less complexity or cross deps